.github/workflows/dev-linux.yml

```diff
@@ -121,12 +121,19 @@ jobs:
       - uses: ./.github/actions/test_stateless_cluster_linux
 
   test_sqllogic_standalone_linux:
-    timeout-minutes: 20
+    timeout-minutes: 15
```

Comment:
Assign a good understanding name like:

```yaml
name: test_sqllogic_standalone_linux_${{ matrix.dirs }}
```

Reply:
Can this name contain a space? For example "test_sqllogic_standalone_linux_gen query" ?

Reply:
why not make it single dir per job?

Reply:
> Can this name contain a space? For example "test_sqllogic_standalone_linux_gen query" ?

I don't think so.

Reply:
Historical Reason, I can move query into gen, and rename gen to databend, Maybe Clearer.

Reply:
> Can this name contain a space? For example "test_sqllogic_standalone_linux_gen query" ?

Yes, it is OK. But not beautiful.